sdk/appconfiguration/app-configuration/test/index.spec.ts

```diff
@@ -28,7 +26,6 @@ describe("AppConfigurationClient", () => {
 
   describe("constructor", () => {
     it("supports connection string", async () => {
-      const client = new AppConfigurationClient(connectionString);
```

Comment:
I actually wanted this to specifically test using the connection string. If we update the tests to use AAD credentials in the future, and forget to add this check back in we'll get a false positive.

Reply:
I will add it back